.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = .git,__pycache__,build,dist,versioneer.py,pcdsdevices/_version.py,pcdsdevices/areadetector/*,docs/*
+exclude = .git,__pycache__,build,dist,pcdsdevices/_version.py,docs/source/conf.py
 max-line-length = 88
```

Comment:
There are a bunch of flake8 that suggest this config wasn't used by GHA.  I need to take a closer look but I imagine we should pull this file in each repo like the pre-commit-config is

Reply:
Hmm, good catch. Odd...

Old: https://github.com/pcdshub/pcds-ci-helpers/blob/ffd48516b17c7ef3ac8fe075a5fc29b1947c1873/travis/pre_commit.sh#L14
New: https://github.com/pcdshub/pcds-ci-helpers/blob/ffd48516b17c7ef3ac8fe075a5fc29b1947c1873/.github/workflows/pre-commit.yml#L90-L93

On the plus side, GHA is _for some reason_ reporting status in this PR unlike when I opened the pcdsutils migration PR.